src/server/game/Loot/LootMgr.h

```diff
@@ -180,24 +180,24 @@ struct TC_GAME_API LootItem
     const AllowedLooterSet & GetAllowedLooters() const { return allowedGUIDs; }
 };
 
-struct QuestItem
+struct NotNormalItem
```

Comment:
Since you are renaming it already, wouldn't it be better if you included the word "Loot" in the name as well?